Clicker/Supporting/Assets.xcassets/upvote-green.imageset/Contents.json

```diff
@@ -6,7 +6,7 @@
     },
     {
       "idiom" : "universal",
-      "filename" : "greyTriangle.png",
+      "filename" : "Triangle 2 (5).png",
```

Comment:
can you rename this to something more descriptive?

Reply:
hmmm, that's weird, I named it to 'upvote-grey' and in the code of the branch, it reflects it.

Reply:
It also seems that you refer to the image later as 'upvote-grey'? So I'm also not sure why the naming is inconsistent..